features/README.md

```diff
@@ -169,11 +169,11 @@ If the rule in the feature file applies only to specific IFC version(s) and/or V
 
 ```
 Given A model with Schema "IFC2X3"
-Given A file with Model View Definition "CoordinationView"
+Given A file with Model View Definition "CoordinationViewV2.0"
```

Comment:
Any reason why  you do not have the _ here?

Reply:
A typo. It's a bit confusing though,[ fixed it](https://github.com/buildingSMART/ifc-gherkin-rules/pull/391/commits/bc9617faaf58a97826ec522122e2b9c256fc81a1).